sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -72,6 +78,7 @@ public LogsQueryClient(Uri endpoint, TokenCredential credential, LogsQueryClient
                 credential,
                 options.AuthenticationScope ?? "https://api.loganalytics.io//.default"));
             _queryClient = new QueryRestClient(_clientDiagnostics, _pipeline, endpoint);
+            Endpoint = endpoint;
```

Comment:
Let's assign the value before we modify it on line 75. So we expose the same value that customers passed in. It would be nice to add a simple test  (here -> https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientTests.cs) that checks that the property value is getting set.